### PR TITLE
WL-18 Added CI github workflows

### PR DESCRIPTION
**Pull request by nagyDarius**

### CODE REVIEW

File: .github/workflows/assemble.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  assemble:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java for build
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Assemble
+        run: gradle assemble
```
Comment (MarioRivis) [inline .github/workflows/assemble.yml]: gradle build -X integrationTest

File: .github/workflows/assemble.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Assemble
```
Comment (MarioRivis) [inline .github/workflows/assemble.yml]: Build

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: Build
```
Comment (MarioRivis) [inline .github/workflows/build.yml:1]: Rename files also -> Release

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    branches:
+      - master
+      - 'release/**'
+  pull_request:
+    branches:
+      - master
+      - 'release/**'
+
+jobs:
+  build:
```
Comment (MarioRivis) [inline .github/workflows/build.yml:12]: job also

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java for build
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build
+        run: gradle build
```
Comment (MarioRivis) [inline .github/workflows/build.yml]: add release to github packages as comment! TODO!!!

**Review by MarioRivis — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 3 above)
Comment (nagyDarius) [reply .github/workflows/build.yml:1]: Renamed to integration test

**Review by MarioRivis — APPROVED** [review]

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build
+        run: gradle build -x integrationTest
```
Comment (MarioRivis) [inline .github/workflows/build.yml:21]: add clean?

File: .github/workflows/integrationTest.yml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  integrationTest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Integration Test
+        run: gradle build
```
Comment (MarioRivis) [inline .github/workflows/integrationTest.yml:23]: add clean?

**Review by MarioRivis — CHANGES_REQUESTED** [review]

**Pull request merged**